h/accounts/views.py

```diff
@@ -400,6 +400,7 @@ class AjaxRegisterController(RegisterController):
 
 
 @view_auth_defaults
+@view_defaults(permission=security.Authenticated)
```

Comment:
This isn't right. `security.Authenticated` is a principal, not a permission. You need an ACL that allows a permission to `Authenticated` on the context and then protect the view with that permission.